non-software.md

```diff
@@ -18,6 +18,14 @@ Any open source software license or open license for media (see [above](#data-me
 
 The [SIL Open Font License 1.1](/licenses/ofl-1.1/) keeps fonts open, allowing them to be freely used in other works.
 
+### Hardware
+
+There are the CERN Open Hardware family of licenses: [CERN-OHL-P-2.0](/licenses/cern-ohl-p-2.0/), the permissive variant ([MIT](/licenses/mit/)-like); [CERN-OHL-W-2.0](/licenses/cern-ohl-w-2.0/), the “weakly reciprocal” variant ([MPL](/licenses/mpl-2.0/)-like), and [CERN-OHL-S-2.0](/licenses/cern-ohl-s-2.0/), the “strongly reciprocal” variant ([GPL](/licenses/gpl-3.0/)-like).  The CERN OHL family is also applicable to HDL source code and synthesized bitstreams as would be used in FPGAs.
+
+[CC-BY-4.0](/licenses/cc-by-4.0/) and [CC-BY-SA-4.0](/licenses/cc-by-sa-4.0/) are also commonly used for hardware; however they do not grant patent rights which can restrict usage of patent-encumbered hardware licensed under them and Creative Commons does not recommend them for hardware.
```

Comment:
I would delete and keep concise with the recommendation not anti-recommendation.